frappe/client.py

```diff
@@ -424,4 +424,21 @@ def validate_link(doctype: str, docname: str):
 			frappe.PermissionError
 		)
 
-	return frappe.db.get_value(doctype, docname, cache=True)
+	values = frappe._dict()
+	values.name = frappe.db.get_value(doctype, docname, cache=True)
+
+	fields = frappe.parse_json(fields)
+	if not values.name or not fields:
+		return values
+
+	try:
+		values.update(get_value(doctype, fields, docname))
+	except frappe.PermissionError:
+		frappe.clear_last_message()
+		frappe.msgprint(
+			_("You need Read permissions to fetch values from {} {}")
+			.format(frappe.bold(doctype), frappe.bold(docname)),
+			title=_("Error Fetching Values")
```

Comment:
```suggestion
			title=_("Failed to fetch values"),
			indicator="orange"
```

Reply:
Error/message titles are generally written in present tense. I understand that we'd like to distinguish this from **Error**. How about something like **Cannot Fetch Values** or **Read Permission Required**?

On a second thought, maybe this should be an error?

Reply:
**Cannot Fetch Values** should work.